ldapadmin/src/test/java/org/georchestra/ldapadmin/ds/AccountDaoTest.java

```diff
@@ -75,5 +84,36 @@ public void testBlankFields_issues_1086_1096() throws Exception {
         assertTrue("Found a 'o' attribute, expeceted none", noOrgAnymore);
         
     }
+
+    @Test
+    public void testUpdateAcountAccount() throws Exception {
```

Comment:
because the method signature is  update(Account old, Account new), and I did not have any imagination for a test name :)